src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -181,6 +183,18 @@ protected void subscribe(boolean shouldAttachHandler) {
         this.consumer.subscribe(topics, this::subscribeHandler);
     }
 
+    /**
+     * Unubscribe all the topics which the consumer currently subscribes
+     *
+     * @param shouldAttachHandler if the handler for getting messages should be set up
+     */
+    protected void unsubscribe(boolean shouldAttachHandler) {
```

Comment:
so the `shouldAttachHandler` is useless, right? remove from the Javadoc as well.

Reply:
Good catch.